resources/codegen/tests/conftest.py

```diff
@@ -0,0 +1,426 @@
+                        id="openassetio-codegen-test-all:namespace.AllProperties",
+                        name="AllProperties",
+                        description="A trait with properties of all types.",
+                        usage=[],
+                        properties=[
+                            datamodel.PropertyDeclaration(
+                                id="boolProperty",
+                                type=datamodel.PropertyType.BOOL,
+                                description="A bool-typed property.",
+                            ),
+                            datamodel.PropertyDeclaration(
+                                id="dictProperty",
+                                type=datamodel.PropertyType.DICT,
+                                description="A dict-typed property.",
+                            ),
+                            datamodel.PropertyDeclaration(
+                                id="floatProperty",
+                                type=datamodel.PropertyType.FLOAT,
+                                description="A float-typed property.",
+                            ),
+                            datamodel.PropertyDeclaration(
+                                id="intProperty",
+                                type=datamodel.PropertyType.INTEGER,
+                                description="A int-typed property.",
+                            ),
+                            datamodel.PropertyDeclaration(
+                                id="stringProperty",
+                                type=datamodel.PropertyType.STRING,
+                                description="A string-typed property.",
+                            ),
+                        ],
+                    ),
+                    datamodel.TraitDeclaration(
+                        id="openassetio-codegen-test-all:namespace.NoProperties",
+                        name="NoProperties",
+                        description="Another trait, this time with no properties.",
+                        properties=[],
+                        usage=[],
+                    ),
+                    datamodel.TraitDeclaration(
+                        id="openassetio-codegen-test-all:namespace.NoPropertiesMultipleUsage",
+                        name="NoPropertiesMultipleUsage",
+                        description="Another trait, this time with multiple usage.",
+                        properties=[],
+                        usage=["entity", "relationship"],
+                    ),
+                ],
+            ),
+        ],
+        specifications=[
+            datamodel.NamespaceDeclaration(
+                id="test",
+                description="Test specifications.",
+                members=[
+                    datamodel.SpecificationDeclaration(
+                        id="LocalAndExternalTrait",
+                        description="A specification referencing traits in this and another package.",
```

Comment:
[consider][minor] 99 char, here and several places below
```suggestion
                        description=(
                            "A specification referencing traits in this and another package."),
```

Reply:
Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/529/commits/45b432e93bce344bb2df64f9c693dae0a74159af).